addons-l10n/en/nfe-checker.json

```diff
@@ -0,0 +1,10 @@
+{
+  "nfe-checker/unreviewed": "Unreviewed",
+  "nfe-checker/title-unreviewed": "The Scratch Team has not looked at this project yet.",
+  "nfe-checker/fe": "Unrestricted",
+  "nfe-checker/title-fe": "The Scratch Team has reviewed this project and deemed it safe for everyone.",
+  "nfe-checker/nfe": "Restricted",
+  "nfe-checker/title-nfe": "This project reviewed by a Scratch Team member and while considered safe for Scratch, won't be available by searching for it and can't appear on the search bar. This rarely happens and it happens with reason. Do not create forum topics or otherwise try to get the Scratch Team to reconsider. If you want to ask why this decision happened, use the Contact Us link in the footer.",
```

Comment:
```suggestion
  "nfe-checker/title-nfe": "This project has been reviewed by a Scratch Team member and was considered okay for Scratch but won't be listed in search results or the Explore page. This rarely happens and it happens with reason. Do not publicly appeal (e.g. create forum topics) or otherwise try to get the Scratch Team to reconsider. If you want to ask why this decision happened, use the Contact Us link in the footer.",
```